.github/ISSUE_TEMPLATE/community_component_proposal.yaml

```
@@ -0,0 +1,22 @@
+name: Community component proposal
+description: Propose a new community component.
+labels: ["community-component"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for opening a community component proposal for Grafana Alloy.
+
+        Before opening the proposal, make sure you read the [Community Component Guidelines](https://github.com/grafana/alloy/tree/main/docs/developer/adding-community-components.md).
```

```suggestion
        Make sure you read the [Community Component Guidelines](https://github.com/grafana/alloy/tree/main/docs/developer/adding-community-components.md).
```

We thank them for opening the proposal and then tell them they need to do a thing BEFORE they open the proposal. We need to either remove the "Before..." part or find another way to phrase the "thank you..." part.